docs/DESIGN.md

```diff
@@ -0,0 +1,24 @@
+# Design Doc
+
+
+## Objective
+
+To Create a new Notification system in TypeScript to provide users with better feeds and more features.
+
+## Goals
+
+Our current notification feed has many limitations. It is specific to a selected archive and doesn’t include all the notifications generated in the system.
+Also our current notifications don’t have actionable buttons which can help users navigate easily in the app and save a lot of time.
+The proposed notification system will be able to eliminate all the limitations of the current notification system and also be able to provide users with better features like actionable buttons.
+
+## Overview
+
+Previously we have been displaying notifications for each archive and hence the notifications feeds were also configured to fetch data based on the archive. Our design has never considered providing users with an ability to take action right from the notification feed. In recent years, new features haven't been rolled out with accompanying notifications, which has left the notifications feed limited in the types of notices it displays to users.
+The new UI design has combined all feeds into one account notification feed, and also has actionable buttons corresponding to each feed. The limitations of the current notification system calls for a redesign and re-architecture. Below you can find the new UI ![design](notification_ui_design.png)
```

Comment:
I have pushed this image file on a previous commit, but Git says it is empty.

Reply:
I saw that empty file in the PR diff too, but it's clearly not correct - maybe a bug in GitHub?

Looking at [DESIGN.md on your branch](https://github.com/PermanentOrg/notification-service/blob/PER-8245/docs/DESIGN.md), the image does show up!

Reply:
Yes I can see it. May be I was looking at the wrong place.